django/core/paginator.py

```diff
@@ -103,12 +102,27 @@ def _check_object_list_is_ordered(self):
                 UnorderedObjectListWarning
             )
 
+    def page_force_into_range(self, number):
```

Comment:
The name for this function is a bit strong, from my perspective. How about `effective_page()`?